src/network-config.js

```diff
@@ -15,6 +18,7 @@ const getNetworkConfig = function getNetworkConfig(networkId) {
                 evmAccountId: 'evm',
                 walletUrl: 'https://wallet.testnet.near.org',
                 explorerUrl: 'https://explorer.testnet.near.org',
+                version: consts.NEAR_NET_VERSION_TEST
```

Comment:
Probably would make sense to rename this constant to `NEAR_NET_VERSION_TESTNET` going forward.